### ISSUE TITLE
In App Changelog

### ISSUE THREAD

**Issue by tmobaird** [tmobaird/i-want-to-contribute#41]
I think this could be a cool feature as well for usage purposes. I'm suggesting that we integrate a changelog for the app into the actual app. I think this should be fairly easy too. This should probably just consist of a page with a list of recent changes/additions that have been made. 

I believe as far as data goes for this, we should probably create a `changelog.json` file in the root of the project. The schema could probably look something like:

```json
{
  "recent": [
    {
      "type": "Bug Fix",
      "title": "Fixing Spelling Error",
      "body": "There was a spelling error on the contributors page, this was updated.",
      "pull_request_id": 120
    },
    {
      "type": "New Feature",
      "title": "Adding Contributors Map",
      "body": "Integrated the contributors map into the actual app using leaflet maps.",
      "pull_request_id": 100
    }
  ]
}
```

This data set could then be parsed and displayed in the app.

This isn't definitive, but here's a list of what I think would need to be done for this:

- [ ] Add `changelog.json`
- [ ] Read `changelog.json` data. Maybe into redux initial state?
- [ ] Create react-router route. (ex. `/changelog`)
- [ ] Create ChangelogPage component
- [ ] Create components to display content

This will be a bit of work, but this is a really good issue for anyone who wants to learn react, react-router, redux through some practical work.

**Comment by kbeaulieu802** [comment]
@tmobaird mind if i work on this?

**Comment by tmobaird** [comment]
@kbeaulieu802 nope! Go for it! 👍 Feel free to let me know if you want any help with this!
